scenarios/crowdsecurity/http-sensitive-files.yaml

```diff
@@ -0,0 +1,17 @@
+type: leaky
+format: 2.0
+debug: true
+name: crowdsecurity/http-sensitive-files
+description: "Detect attempt to access to sensitive files (.log, .db ..) or folders (.git)"
+filter: 'evt.Meta.service == "http" and any(File("sensitive_data.txt"), { evt.Parsed.request endsWith #})'
```

Comment:
as well I think the filter is too lose, we should check that it's access or error log no ?

Reply:
done :+1: